core/workspace_svg.js

```diff
@@ -944,6 +944,7 @@ Blockly.WorkspaceSvg.prototype.loadAudio_ = function(filenames, name) {
     if (ext && audioTest.canPlayType('audio/' + ext[1])) {
       // Found an audio format we can play.
       sound = new window['Audio'](filename);
+      sound.setAttribute('preload', 'auto');
```

Comment:
This change breaks iOS.  The existing solution will ensure that the click sound plays, something we want to preserve.  I've tried several variations of this PR on iOS but have not yet been able to come up with something that plays any sound.